gapic/src/main/com/google/gapid/perfetto/views/TraceComposite.java

```diff
@@ -300,15 +306,18 @@ private static int permyriad(long v, long t) {
   private static class TopBar extends Composite {
     private final ToolBar toolBar;
 
-    public TopBar(Composite parent, Analytics analytics, Theme theme) {
+    public TopBar(Composite parent, Models models, Theme theme) {
       super(parent, SWT.NONE);
       setLayout(new GridLayout(3, false));
```

Comment:
Since you added a new icon, the 3 should go to 4 here, denoting 4 column grid layout to make both icons on the same row.